include/caffe/layers/batch_norm_layer.hpp

```diff
@@ -7,6 +7,9 @@
 #include "caffe/layer.hpp"
 #include "caffe/proto/caffe.pb.h"
 
+#define BN_VARIANCE_CLIP_START 200
+#define BN_VARIANCE_CLIP_CONST 4.0
```

Comment:
I'll think later about what these constants are doing, but for now: don't use `#define` when you could use a `const` variable.